src/python/system/process_handler.py

```diff
@@ -144,7 +144,8 @@ def run_process(cmdline,
   # This is used when running scripts on native linux OS and not on the device.
   # E.g. running a fuzzer to generate testcases or launcher script.
   plt = environment.platform()
-  if plt in ['ANDROID', 'FUCHSIA'] and (not testcase_run or launcher):
+  is_android = environment.is_android(plt)
```

Comment:
nit: can we do something like

```python
runs_on_device = environment.is_android(plt) or plt == 'FUCHSIA'
if runs_on_device and (not testcase_run or launcher):
 ...
```

I think that's a bit clearer?

Reply:
Done!